addon/styles/addon.css

```diff
@@ -96,3 +96,30 @@
 .sf-files-card--small > div.au-u-margin-top-tiny {
   margin-top: 0 !important;
 }
+
+.center-container {
+  display: flex;
+  align-items: center;
+}
+
+.input-clear-btn {
+  cursor: pointer;
+  width: 2rem;
+  height: 2rem;
+  color: var(--au-gray-700);
+  background-color: var(--au-gray-200);
+  border-radius: 100%;
+  margin-left: -2.5rem;
+}
+
+.number-input-no-spin::-webkit-inner-spin-button,
```

Comment:
i don't know if we want to remove this. E.g. on mobile it could be very nice to have in some cases, no?

Reply:
I only removed it if you have the option allowClear set to true, so in other cases it is still there. Seemed better UI wise, but I can certainly revert this commit.